spatial-tracking-explainer.md

```diff
@@ -246,18 +242,10 @@ const RAD_PER_PIXEL = Math.PI / 180.0; // (1 degree)
 function onPointerMove(event) {
   let s = Math.sin(event.movementX * RAD_PER_PIXEL);
   let c = Math.cos(event.movementX * RAD_PER_PIXEL);
-  let o = xrReferenceSpace.originOffset.orientation;
-
-  xrReferenceSpace.originOffset = new XRRigidTransform(
-      // Keep the previous position
-      xrReferenceSpace.originOffset.position,
-      // Quaternion math to rotate the previous orientation around the Y axis.
-      {
-        x: o.x * c - o.z * s,
-        y: o.y * c + o.w * s,
-        z: o.z * c + o.x * s,
-        w: o.w * c - o.y * s
-      });
+
+  // Computes a quaternion to rotate around the Y axis.
+  xrReferenceSpace = xrReferenceSpace.getOffsetReferenceSpace(
+    new XRRigidTransform(null, { x: 0, y: s, z: 0, w: c }));
```

Comment:
I think this example may be a bit confusing, it implies that it rotates by a given angle, but the usual "quaternion from axis angle" formulas use sin/cos of half angles internally. I guess it doesn't matter since it's based on an arbitrary scale factor, but could lead to wrong results if people generalize it.

Reply:
Fixed